LEGO1/lego/sources/anim/legoanim.h

```diff
@@ -24,6 +24,17 @@ class LegoAnimKey {
 	LegoU32 TestBit2() { return m_flags & c_bit2; }
 	LegoU32 TestBit3() { return m_flags & c_bit3; }
 
+	// FUNCTION: BETA10 0x100739a0
+	void FUN_100739a0(MxS32 p_param)
+	{
+		if (p_param) {
+			m_flags |= 0x01;
+		}
+		else {
+			m_flags &= 0xfe;
```

Comment:
I'd introduce an enum constant for this flag in the class (see other enums for examples). Clearing the flag is then done with `&= ~c_bit1`